packages/next-swc/crates/next-core/src/app_segment_config.rs

```diff
@@ -431,6 +436,14 @@ fn parse_config_value(
 
             config.preferred_region = Some(preferred_region);
         }
+        // Match exported generateImageMetadata function and generateSitemaps function, and pass
+        // them to config.
```

Comment:
That won't work for all cases. `parse_config_value` is only called for var declarations, but not for function declarations.